### DISCUSSION TITLE
what happens to the enchantments

### DISCUSSION THREAD

**Discussion by DKLYNTLY** [RobertSkalko/Age-of-Exile#70] [Q&A]
when u enchant a level 30 diamond sword with knockback do the enchantments disappear?

**Comment by RobertSkalko** [comment]
Did you try pressing CTRL to view enchants

**Reply by DKLYNTLY** [reply]
yes it didnt work

**Reply by RobertSkalko** [reply]
CTRL views normal vanilla tooltip which includes enchants. If still not there, then you didn't actually enchant the item
